### DISCUSSION TITLE
AOSP 11.0 v310

### DISCUSSION THREAD

**Discussion by phhusson** [phhusson/treble_experimentations#1931] [General]
- July security patch level

<hr /><em>This discussion was created from the release <a href='https://github.com/phhusson/treble_experimentations/releases/tag/v310'>AOSP 11.0 v310</a>.</em>

**Comment by davigamer987** [comment]
noice

**Reply by manto35070** [reply]
Somebody have flashed on Xiaomi Redmi 4X ? Which  vendor do you have used ? Thank you in advance, Aldo.

**Reply by Jfuchs200** [reply]
Is there one without the bottom google search bar?

**Comment by Mahamadou1973** [comment]
Bjr. Est-ce-que magisk fonctionne sur cette version ? Si oui comment faire pour l'installer ? Sur les anciennes versions j'ai essayer en vain.

**Comment by nothingnesses** [comment]
Edit: Nevermind, I guess [version 10 and presumably also 11 are incompatible with Magisk](https://github.com/phhusson/treble_experimentations/wiki/Moto-G6-Play), which I had also flashed.

Original: Flashing this version of system-roar-arm-aonly-vanilla.img on my Moto G6 Play (model XT1922-2) device using TWRP v3.2.1-r1 (the latest version of TWRP I could find that works for my device) seems to cause boot loops. The last working version seems to be system-arm-aonly-vanilla-nosu.img [AOSP 9.0 v123](https://github.com/phhusson/treble_experimentations/releases/tag/v123) which I can re-flash to get my device working again. How might I get AOSP 11 to work on my device?

**Comment by BlakeU666** [comment]
My phone went into a bootloop, even though I installed the correct GSI. I have Redmi Note 9 Pro Global (or just joyeuse/miatoll) and the GSI which I tried is system-roar-arm64-aonly-gapps.img. I checked Treble Check and it said that my phone doesn't support seamless updates so it's A-Only and the architecture is aarch64. What did I do wrong?

Oh, also, I installed it using fastboot, since I have super partition, so I can't flash it through custom recovery.

**Comment by phhusson** [comment]
That's not the correct image.

19 juil. 2021 21:56:35 BlakeU666 ***@***.***>:

**Reply by BlakeU666** [reply]
Which one should I choose then?

**Reply by BlakeU666** [reply]
I also tried A/B and it doesn't work as well. I doubt other architecture will work since I am ultra sure my phone has arm64.

**Comment by BlakeU666** [comment]
Hello? Which image should I choose?

**Reply by shansaju** [reply]
Depending on device and vendor

**Comment by DiogoCosta35** [comment]
I installed it on my moto g7 play and it's giving a written error: THIS DEVICE IS NOT CERTIFIED BY PLAY PROTECT.

**Reply by phhusson** [reply]
Register your device on https://www.google.com/android/uncertified/

To get the ID, you can either:
- adb root, then adb shell, press enter, and at the new prompt, inside adb shell, do sqlite3 /data/data/com.google.android.gsf/databases/gservices.db    'select * from main where name = "android_id";'
- https://play.google.com/store/apps/details?id=com.evozi.deviceid => Google Service Framework (GSF) ID
- for multiuser see https://github.com/phhusson/treble_experimentations/issues/1794#issuecomment-816632811

**Reply by shansaju** [reply]
Do you believe it works because I tried many times it doesn't worked for me 😔

**Comment by turuncu2011** [comment]
I'm installing AOSP 11.0 v310, the navigation bar, status bar and sound panel do not open. Also, when I restart, the black screen does not open, what is the problem?

**Comment by dev-zero** [comment]
Awesome, thanks a lot for this! I am using `system-roar-arm64-ab-floss.img` on a Huawei P10 (switched from a MIUI 9.1) and the basics seem to work. What doesn't work is:

* any sound file I'm trying to play (as ringtone or via a music player) does not play and instead of the name I get `android.resource://com.google.android....`, also it seems that the speaker is not working (tried during a call)?
* trying to flash the `system-roar-arm64-ab-vndklite-floss.img.xz` fails with a `Invalid sparse file format at header magic` at the beginning and bails out in the first step. The reason I want to use the vndklite is because it is supposed to give you a rw root, which seem to need to to "unroot" my phone (resp. keeping it rooted with Magisk, I am trying to use phh Magisk Canary v23). Maybe of importance, I already tried to install Magisk by a) flashing a patched `recovery_ramdisk` and b) in the app via the `Direct Install`. Not sure whether this may cause the problem with the vndklite image now.
* signal meter is not working (for none of the provided options)
* Silence immediately crashes with `InvalidKeyException`

**Comment by turuncu2011** [comment]
AOSP 11.0 v311, status bar, nav bar and sound panel are still missing. Why don't you explain a solution for this?

**Reply by turuncu2011** [reply]
APPEAL YOU DON'T TAKE US SO SERIOUSLY.

**Comment by retrocountry** [comment]
Does this work on a Galaxy S9 With SafeStrap Extreme Syndicate?

[Extreme Syndicate](https://xdaforums.com/t/2022-root-extreme-syndicate-g960u-u1-w-g965u-u1-w-n960u-u1-w.4041815/)

**Reply by retrocountry** [reply]
Here's my compatibilty list.

![image](https://github.com/phhusson/treble_experimentations/assets/76595690/532cb9e0-214a-415a-b29b-5e6122be45d4)
